Improve documentation v4 (LC15) (#4973)
Deletescape-Media/Lawnchair: systemUI/README.md

# SystemUI Module

> This directory contains all of the required SystemUI modules, such as Common, which serves as a helper for other SystemUI modules like Unfold for handling devices with hinges.

Reviewer: ```suggestion
This directory contains all of the required SystemUI modules. Some examples of modules include `Common`, which serves as a helper for other SystemUI modules, and `Unfold`, which handles devices with hinges.
```

> See: https://android.googlesource.com/platform/frameworks/base/+/refs/heads/main/packages/SystemUI/

Reviewer: ```suggestion
Upstream source code: https://android.googlesource.com/platform/frameworks/base/+/refs/heads/main/packages/SystemUI/
```
